css/frontend/src/main/scala/io/udash/css/CssView.scala

```diff
@@ -0,0 +1,80 @@
+package io.udash.css
+
+import io.udash.ReadableProperty
+import io.udash.properties.PropertyCreator
+import org.scalajs.dom
+import org.scalajs.dom.Element
+
+import scalatags.JsDom.all.Modifier
+import scalatags.text.Builder
+
+/** Contains integration of CSS structures with Scalatags. */
+trait CssView {
```

Comment:
Extract implicit classes to companion object and provide a conversion here. This will allow you to use value classes.